app/views/content/home.md

```diff
@@ -5,8 +5,8 @@
     Explore how to get into teaching with official Department for Education guidance on training courses, finding funding, and what teaching is really like.
   fullwidth: true
   date: "2023-04-19"
-  title_paragraph: "Nobody knows teaching like we do. Whether you're just thinking about it or ready to apply, we offer free advice and support to decide if teaching in a primary or secondary school in England is right for you."
-  image: "static/content/hero-images/0012.jpg"
+  title_paragraph: "Nobody knows teaching like we do. Whether you're just thinking about it or ready to apply, we offer <strong>free advice and support</strong> to decide if <strong>teaching in a primary or secondary school</strong> in England is right for you."
```

Comment:
@garethwilliamsdesign When you look at this again with Pavel, I wonder if it is worth increasing the line height/adding padding in this section as it feels a bit cramped to read compared with the rest of the site (suspect you've already noticed this!)?

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/5f8ce54c-9416-4325-a98a-8e5b55d5de53)